File: solutions/1236-n-th-tribonacci-number/solution.py
class Solution:
    def tribonacci(self, n: int) -> int:
        tribonakki=[]
        tribonakki.append(0)
        tribonakki.append(1)
        tribonakki.append(1)
        for i in range(3,n+1) :
            tribonakki.append(tribonakki[i-1]+tribonakki[i-2]+tribonakki[i-3])
        
        return tribonakki[n]  

File: solutions/1236-n-th-tribonacci-number/solution.c
static int tribonakki[38];

int tribonacci(int n){
    tribonakki[0]=0;
    tribonakki[1]=1;
    tribonakki[2]=1;
    for(int i=3;i<38;i++){
        tribonakki[i]=tribonakki[i-1]+tribonakki[i-2]+tribonakki[i-3];
    }
    return tribonakki[n];
};

